cfgov/v1/models/molecules.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailimages.blocks import ImageChooserBlock
+
+from .base import CFGOVPage
+
+
+class HalfWidthLinkBlob(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=False)
```

Comment:
[GHE]/flapjack/Modules-V1/wiki/Half-Width-Link-Blob says headings should be required.